show/main.py

```diff
@@ -2081,6 +2081,98 @@ def ztp(status, verbose):
+                             values["local_addr"], values["local_asn"], values["local_port"], values["sent_cap"], values["recv_cap"]])
+
+    click.echo(tabulate(bmp_body, bmp_headers))
+
+# 'bmp-rib-out-table' subcommand ("show bmp bgp-rib-out-table")
+@bmp.command('bgp-rib-out-table')
+@clicommon.pass_db
+def bmp_rib_out_table(db):
+    """Show bmp bgp-rib-out-table information"""
+    bmp_headers = ["Neighbor_Address", "NLRI", "Origin", "AS_Path", "Origin_AS", "Next_Hop", "Local_Pref",
+                   "Originator_ID",  "Community_List", "Ext_Community_List"]
+
+    bmp_keys = db.db.keys(db.db.STATE_DB, "BGP_RIB_OUT_TABLE|*")
+
+    click.echo("Total number of bmp bgp-rib-out-table: {}".format(0 if bmp_keys is None else len(bmp_keys)))
+
+    bmp_body = []
+    if bmp_keys is not None:
+        for key in bmp_keys:
+            key_values = key.split('|')
+            values = db.db.get_all(db.db.STATE_DB, key)
+            bmp_body.append([key_values[3], key_values[1], values["origin"], values["as_path"], values["origin_as"], values["next_hop"], values["local_pref"], values["originator_id"],
+                             values["community_list"], values["ext_community_list"]])
+
+    click.echo(tabulate(bmp_body, bmp_headers))
+
+# 'bgp-rib-in-table' subcommand ("show bmp bgp-rib-in-table")
+@bmp.command('bgp-rib-in-table')
+@clicommon.pass_db
+def bmp_rib_in_table(db):
+    """Show bmp bgp-rib-in-table information"""
+    bmp_headers = ["Neighbor_Address", "NLRI", "Origin", "AS_Path", "Origin_AS", "Next_Hop", "Local_Pref",
+                   "Originator_ID",  "Community_List", "Ext_Community_List"]
+
+    bmp_keys = db.db.keys(db.db.STATE_DB, "BGP_RIB_IN_TABLE|*")
+
+    click.echo("Total number of bmp bgp-rib-in-table: {}".format(0 if bmp_keys is None else len(bmp_keys)))
+
+    bmp_body = []
+    if bmp_keys is not None:
+        for key in bmp_keys:
+            key_values = key.split('|')
+            values = db.db.get_all(db.db.STATE_DB, key)
+            bmp_body.append([key_values[3], key_values[1], values["origin"], values["as_path"], values["origin_as"], values["next_hop"], values["local_pref"], values["originator_id"],
+                             values["community_list"], values["ext_community_list"]])
+
+    click.echo(tabulate(bmp_body, bmp_headers))
+
+# 'tables' subcommand ("show bmp tables")
+@bmp.command('tables')
+@clicommon.pass_db
+def tables(db):
+    """Show bmp table status information"""
+    bmp_headers = ["Table_Name", "Enabled"]
+    bmp_body = []
+    click.echo("BMP tables: ")
+    try:
+        bmp_keys = db.cfgdb.get_table('BMP')
+        if bmp_keys['table']:
+            bmp_body.append(['bgp_neighbor_table', bmp_keys['table']['bgp_neighbor_table']])
+            bmp_body.append(['bgp_rib_in_table', bmp_keys['table']['bgp_rib_in_table']])
+            bmp_body.append(['bgp_rib_out_table', bmp_keys['table']['bgp_rib_out_table']])
+    except KeyError:
```

Comment:
>KeyError [](http://example.com/codeflow?start=11&length=8)

Why you expected there is chance of this exception? If yes, could you add testcase to cover this line?